app/graphql/resolvers/webhooks_resolver.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Resolvers
+  class WebhooksResolver < Resolvers::BaseResolver
+    include AuthenticableApiUser
+    include RequiredOrganization
+
+    description 'Query Webhooks'
+
+    argument :page, Integer, required: false
+    argument :limit, Integer, required: false
+    argument :status, Types::Webhooks::StatusEnum, required: false
+
+    type Types::Webhooks::Object.collection_type, null: false
+
+    def resolve(page: nil, limit: nil, status: nil)
+      validate_organization!
+
+      webhooks = current_organization.webhooks
```

Comment:
It would be good to switch to query service (WebhooksQuery), especially if there will be API endpoint for fetching webhooks

Reply:
It will be on an other PR !